site-src/geps/gep-1619.md

```diff
@@ -0,0 +1,89 @@
+# GEP-1619: Session Persistence and Session Affinity
+
+* Issue: [#1619](https://github.com/kubernetes-sigs/gateway-api/issues/1619)
+* Status: Provisional
+
+(See definitions in [GEP Status][/contributing/gep#status].)
+
+## TLDR
+
+This GEP proposes a definition of session persistence and session affinity as the foundation of future session persistence related improvements. This GEP also outlines the ways session persistence could be achieved by implementations.
+
+## Goals
+- Define session persistence and session affinity to establish a common language
+- Identify differences in session persistence functionality between implementations
+
+## Non-Goals
+- Create proposals for adding session persistence configuration to Gateway API object specification
+- Mandate a default or expected session persistence functionality for implementations
+- Require implementations to support session persistence or a specific form of it
+
+## Introduction
+
+At the November 28th, 2022 SIG Gateway API meeting, we discussed the idea of standardizing and documenting session persistence (also known as session stickiness). It was recommended that we first generate discussion for how we describe session persistence, agree on how it is achieved, and find out which implementers support session persistence. 
+
+Conformance test for session persistence will be considered if deemed appropriate. Furthermore, this information can be used for developing a Gateway API spec to configure session persistence in the future.
```

Comment:
Instead of providing a history, I think I'd just explain what session affinity means and why it should be configurable within Gateway API (assuming that's the goal). Edit: Looks like you've already done that well below, so maybe just leave this part out.